Multisig/UI/Safe Management/Create Safe/CreateSafeFlow.swift

```diff
@@ -68,12 +77,29 @@ class CreateSafeFlow: UIFlow {
     }
 
     func appleLogin() {
-        // TODO: Create login via apple
-        // This line is not needed after this method implementation
-        creatingSafe()
+        handleAuthorizationAppleIDButtonPress()
+
         // TODO: submit create safe tx
     }
 
+    func handleAuthorizationAppleIDButtonPress() {
+        let appleIDProvider = ASAuthorizationAppleIDProvider()
+        let request = appleIDProvider.createRequest()
+        request.requestedScopes = [.fullName, .email]
+
+        appleWeb3AuthLogin = AppleWeb3AuthLogin {
+            LogService.shared.debug("onClose called....")
```

Comment:
debug message left here.. will this code area be worked on further?

Reply:
removed.